### PR TITLE
[full-ci] [tests-only] Removed core bug demonstration scenarios

### PR DESCRIPTION
**Pull request by grgprarup**
## Description
Forward porting of PR https://github.com/owncloud/ocis/pull/5485

## Related Issue

- Part of https://github.com/owncloud/ocis/issues/5337

## How Has This Been Tested?

- :robot: 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/coreApiWebdavLocks/exclusiveLocks.feature [hunk 1]
```diff
@@ -66,7 +66,7 @@ Feature: there can be only one exclusive lock on a resource
       | new      | shared     |
       | new      | exclusive  |
 
-  @skipOnOcV10 @issue-34358 @notToImplementOnOCIS
+  @notToImplementOnOCIS
```
Comment (saw-jan) [inline tests/acceptance/features/coreApiWebdavLocks/exclusiveLocks.feature:69]: you have removed @notToImplementOCIS test, also remove here (?)

File: tests/acceptance/features/coreApiWebdavLocks/requestsWithToken.feature [hunk 2]
```diff
@@ -63,7 +63,7 @@ Feature: actions on a locked item are possible if the token is sent with the req
       | new      | shared     |
       | new      | exclusive  |
 
-  @skipOnOcV10 @issue-34338 @files_sharing-app-required @notToImplementOnOCIS
+  @files_sharing-app-required @notToImplementOnOCIS
```
Comment (saw-jan) [inline tests/acceptance/features/coreApiWebdavLocks/requestsWithToken.feature:66]: same here remove the whole scenario

File: tests/acceptance/features/coreApiWebdavUpload2/uploadFileToBlacklistedNameUsingOldChunking.feature [hunk 3]
```diff
@@ -9,20 +9,20 @@ Feature: users cannot upload a file to a blacklisted name using old chunking
     And using old DAV path
     And user "Alice" has been created with default attributes and without skeleton files
 
-  @skipOnOcV10 @issue-36645 @notToImplementOnOCIS
+  @notToImplementOnOCIS
```
Comment (saw-jan) [inline tests/acceptance/features/coreApiWebdavUpload2/uploadFileToBlacklistedNameUsingOldChunking.feature:12]: scenario to remove

File: tests/acceptance/features/coreApiWebdavLocks/exclusiveLocks.feature (hunk 1 above)
Comment (grgprarup) [reply tests/acceptance/features/coreApiWebdavLocks/exclusiveLocks.feature:69]: We have separate PR to remove `@notToImplementOnOcis`

**Review by saw-jan — COMMENTED** [review]
:+1:

**Review by saw-jan — APPROVED** [review]

**Pull request merged**